test/conformance/kuttl/mutate/clusterpolicy/cornercases/mutate-using-default-context/01-pod.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- file: pod.yaml
+assert:
+- pod-assert.yaml
```

Comment:
You want to assert the error when applying pod.yaml, not the other pod. Something like this:
```
apply:
- file: pod.yaml
  shouldFail: true
```

Reply:
done